github/Requester.py

```diff
@@ -67,19 +67,89 @@
 atLeastPython3 = sys.hexversion >= 0x03000000
 
 
+class RequestsResponse:
+    # mimic the httplib response object
+    def __init__(self, r):
+        self.status = r.status_code
+        self.headers = r.headers
+        self.text = r.text
+
+    def getheaders(self):
+        if atLeastPython3:
+            return self.headers.items()
+        else:
+            return self.headers.iteritems()
```

Comment:
So `viewitems` is a bit more accurate/similar to the Python 3 behavior on Python 2.

Reply:
True, although this works on python2 ;-) . I think the only difference is performance, no?

Reply:
They would both return a `dict_items` object. That would be the main motivator in my mind at least.

Reply:
Since the only place this will be used is:
```
responseHeaders = dict((k.lower(), v) for k, v in response.getheaders())
```
I think the type doesn't really matter.